lib/src/components/NxVulnerabilityIndicator/NxVulnerabilityIndicator.tsx

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import classnames from 'classnames';
+
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxVulnerabilityIndicator.scss';
+
+export default function NxVulnerabilityIndicator({
+  criticalCount, severeCount, moderateCount, lowCount, layout, className, ...attrs }: Props) {
+
+  const layoutClasses = classnames('nx-vulnerability-indicator-container', className, {
+    [`nx-vulnerability-indicator-container--${layout}`]: layout,
+    ['nx-vulnerability-indicator-container--row']: !layout
+  });
+
+  return (
+    <div className={layoutClasses} {...attrs}>
+      <div className="nx-vulnerability-indicator nx-vulnerability-indicator--critical">
+        <span className="nx-vulnerability-indicator__count">{criticalCount}</span>
+        <span className="nx-vulnerability-indicator__text">Critical</span>
```

Comment:
If you want to simplify the DOM slightly, the text styles for this element could be moved to its parent (and the margin moved over to the count).  It's also fine as-is though

Reply:
> If you want to simplify the DOM slightly, the text styles for this element could be moved to its parent (and the margin moved over to the count). It's also fine as-is though

I get what you're saying, but I prefer having the font styles directly on the thing to which they apply.